src/diffusers/utils/torch_utils.py

```diff
@@ -286,6 +286,8 @@ def get_device():
         return "xpu"
     elif torch.backends.mps.is_available():
         return "mps"
+    elif torch.mlu.is_available():
```

Comment:
```suggestion
    elif is_mlu_available:
```
also need to import this